packages/builder/src/pages/builder/app/[application]/data/datasource/[datasourceId]/_components/panels/Tooltip.svelte

```diff
@@ -0,0 +1,30 @@
+<script>
+  import FontAwesomeIcon from "components/common/FontAwesomeIcon.svelte"
+  import { Body } from "@budibase/bbui"
+
+  export let title = ""
+  export let href = null
+</script>
+
+<a {href} class="tooltip" target="_blank" rel="noopener noreferrer">
```

Comment:
After speaking with Andy (because I know it looks similar to this in the docs) could we update this to be a quiet action button instead?

Code would basically just be
```js
<ActionButton quiet icon="Help" on:click={() => window.open(href, "_blank")}>
  {title}
</ActionButton>
```